activities/events/deciding-to-attend-events.md

```diff
@@ -17,12 +17,13 @@ We have different decision making criteria for these 2 types of event attendance
 
 When considering whether to attend an event, you must answer:
 
-How does sending a delegation to an event further our [mission](https://about.publiccode.net/organization/mission.html)?
+How does sending a delegation to an event further our [mission](../../organization/mission.md)?
 
 It might help to explain:
 
 * the goals the organization is trying to achieve by attending the event, including how you'll measure if the goals were met
 * which and how many people to send to make sure the goals are met
+* also beware that [communication efforts](../communication/), while not directly being part of the mission, is an enabler for it
```

Comment:
I don't understand what this sentence tries to say

Reply:
Agreed, it isn't super clear. What I wanted to achieve was the thing we highlighted about visibility for the Foundation. It's not really in our mission to be noticed, but of course no one will join us if we aren't known for being trustworthy experts. Do you have a suggestion how to make that more clear?

Reply:
Hey, I've rewritten this sentence so it's more clear.